src/components/AnnotationBrowser/AnnotationList.vue

```diff
@@ -18,20 +18,45 @@
         </v-row>
         <v-row>
           <v-col class="pa-0 mx-1">
-            <v-dialog v-model="tagSelectedDialog" width="50%">
+            <v-dialog v-model="addRemoveTagsSelectedDialog" width="50%">
```

Comment:
This v-model name really hides what it does  
What about renaming it to  `showAddRemoveTagsSelectedDialog`